tests/generation/test_utils.py

```diff
@@ -2884,6 +2885,42 @@ def test_return_unprocessed_logit_scores(self):
         self.assertTrue(y_prob > 0.001 and n_prob > 0.001)
         self.assertTrue(y_prob <= 1.0 and n_prob <= 1.0)
 
+
+@require_torch
+class TokenHealingTestCase(unittest.TestCase):
+    @parameterized.expand(
+        [
+            (
+                "square_bracket",
+                'An example ["like this"] and another example [',
+                'An example ["like this"] and another example ["',
+            ),
+            ("url", 'The link is <a href="http:', 'The link is <a href="http://'),
+            ("aggressive_healing", 'The link is <a href="http', 'The link is <a href="http'),
```

Comment:
Why is this one called `agressive_healing`? The input and the expected result are the same, no?

Reply:
Thanks for getting back! I added back the two files and a comment explanation in the new push :)